Makefile

```diff
@@ -13,13 +13,13 @@ OPERATOR_IMAGE ?= cluster-network-addons-operator
 REGISTRY_IMAGE ?= cluster-network-addons-registry
 export OCI_BIN ?= $(shell if podman ps >/dev/null 2>&1; then echo podman; elif docker ps >/dev/null 2>&1; then echo docker; fi)
 TLS_SETTING := $(if $(filter $(OCI_BIN),podman),--tls-verify=false,)
-PLATFORMS ?= linux/amd64
+ARCH := $(shell uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
+PLATFORMS ?= linux/${ARCH}
```

Comment:
wdyt about adding an option to set `export PLATFORMS=all` 
which will be translated internally to `linux/amd64,linux/arm64,linux/s390x` ?
it is less explicit but easier to maintain

Reply:
@oshoval That's a great idea! Adding an `export PLATFORMS=all` option would simplify the configuration and make it easier to maintain, especially as we add more platforms in the future. I'll include this change in a follow-up PR.
All tests have passed, and there is one pending tide test awaiting the approval label.

Reply:
Updated PLATFORMS to automatically include all supported platforms when export PLATFORMS=all is used.